packages/odata-common/src/request-builder/batch/batch-request-builder.ts

```diff
@@ -103,6 +113,31 @@ export class BatchRequestBuilder<
     });
     return request;
   }
+
+  private getOperationDeserializer(
```

Comment:
[req/pp] As this returns (de-)**S**erialiser**S** I would suggest to name it that 😊 Note that those are serializers and deserializers and therefore `deSerializers` not `deserializers`.
```suggestion
  private getOperationDeSerializers(
```

Reply:
applied